sarracen/sarracen_dataframe.py

```diff
@@ -3,6 +3,8 @@
 
 class SarracenDataFrame(DataFrame):
 
+    _metadata = ['_params', 'params']
```

Comment:
I think we can get away here with just
`use _metadata = ['_params']`
A bit simpler. Can then remove the self.params = dict() line below.